File: Cinemabe/Cinema/Data/Models/Movie.cs
namespace Cinema.Data.Models
{
    public class Movie
    {
        public Guid Id { get; set; }
		public Guid AgeRestrictionId { get; set; }
		public AgeRestriction AgeRestriction { get; set; }
        public Guid ShowTimeTypeId { get; set; }
        public ShowTimeType ShowTimeType { get; set; }
        public string Name { get; set; }
        public string Image { get; set; }
        public int Time { get; set; }
        public DateTime ReleaseDate { get; set; }
        public string Description { get; set; }
        public string Director { get; set; }
        public string Actor { get; set; }
        public string Trailer { get; set; }
        public string Languages { get; set; }
        public bool Status { get; set; }
    }
}

File: cinema_app/lib/models/movie.dart
class Movie {
  int id;
  String name;
  int time;
  int ageRestrictionId;
  DateTime releaseDate;
  String description;
  String director;
  String actor;
  String trailer;
  String languages;
  String img;
  Movie(
      {required this.id,
      required this.actor,
      required this.ageRestrictionId,
      required this.description,
      required this.director,
      required this.img,
      required this.languages,
      required this.name,
      required this.releaseDate,
      required this.time,
      required this.trailer});
}
